python/lsst/afw/table/Base.i

```diff
@@ -609,6 +610,7 @@ aliases = {
     int: "I",
     float: "D",
     str: "String",
+    futurestr: "String",
```

Comment:
I'm confused why this is needed. Can't we just use `from builtins import str` everywhere and forget about the old `str`?